src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -454,14 +461,10 @@ internal sealed class RSession : IRSession, IRCallbacks {
         /// Graph app may call Win32 API directly rather than going via R API callbacks.
         /// </summary>
         /// <returns>Pressed button code</returns>
-        async Task<MessageButtons> IRCallbacks.ShowDialog(IReadOnlyList<IRContext> contexts, string s, bool isEvaluationAllowed, MessageButtons buttons, CancellationToken ct) {
+        async Task<MessageButtons> IRCallbacks.ShowDialog(IReadOnlyList<IRContext> contexts, string s, MessageButtons buttons, CancellationToken ct) {
```

Comment:
So evaluation can be requested when host is in one state and get received when it is in the different state?